<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YT-DLP GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ytdlp.png</normaloff>ytdlp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Youtube URL/ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ytdlp.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pathButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Download Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="tableLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Downloads:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="downloadTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>871</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Completion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Speed</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Downloaded</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ETA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Elapsed</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="optionsButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QComboBox" name="formatBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Video</string>
    </property>
    <property name="placeholderText">
     <string>Select Format</string>
    </property>
    <item>
     <property name="text">
      <string>Video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   mp4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   mkv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   flv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   webm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   mov</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   avi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   mka</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   ogg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   aac</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Audio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   mp3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   aac</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   m4a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   opus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   vorbis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   flac</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   alac</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   wav</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>   webm</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
